client/state/application/actions.js

```diff
@@ -35,3 +47,36 @@ export function connectionRestored( noticeText ) {
 		dispatch( { type: CONNECTION_RESTORED } );
 	};
 }
+
+export function cssBuildFailed( noticeText ) {
+	return ( dispatch ) => {
+		dispatch( removeNotice( 'cssBuilding' ) );
+		dispatch( errorNotice(
+			noticeText
+			, {
```

Comment:
No problem. I had modeled the last two functions from the existing first two so I've fixed all 4 of them.